copyright

```diff
@@ -3615,6 +3616,139 @@ Copyright: NASA
 License: public-domain
  Derived from works by NASA/JPL-Caltech/Malin Space Science Systems.
 
+Files:
+ images/effect/successor?flare/*
+ images/effect/bimodal?impact*
+ images/effect/bimodal?switch?impact*
+ images/effect/drag?cannon?impact*
+ images/effect/overcharged?pulse?laser?impact*
+ images/effect/successor?pulse?laser?impact*
+ images/effect/void?spark*
```

Comment:
```suggestion
 images/effect/bimodal?impact*
 images/effect/bimodal?switch?impact*
 images/effect/drag?cannon?impact*
 images/effect/overcharged?pulse?laser?impact*
 images/effect/successor?flare/*
 images/effect/successor?pulse?laser?impact*
 images/effect/void?spark*
```
Make sure these are in alphabetical order. Reflect this in the DPI repo too if applicable.